taxi/views.py

```diff
@@ -28,6 +28,21 @@
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def update_drivers(request, pk: int):
+    car = Car.objects.get(id=pk)
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+        car.save()
+    else:
+        car.drivers.remove(request.user)
+        car.save()
```

Comment:
The logic for adding or removing a user from the `drivers` field could be improved by checking if the user is already associated with the car before attempting to add or remove them. This would prevent unnecessary database operations.